Client/qtTeamTalk/utilui.cpp

```diff
@@ -177,6 +177,10 @@ void migrateSettings()
             ttSettings->remove("texttospeech/tts-timestamp");
         }
 #endif
+
+        // Default sounds pack name changed in 5.5 format
+        if (ttSettings->value(SETTINGS_SOUNDS_PACK, SETTINGS_SOUNDS_PACK_DEFAULT).toString() == QCoreApplication::translate("UtilUI", "Default"))

```

Comment:
To disable the translation shouldn't it be:
`if (ttSettings->value(SETTINGS_SOUNDS_PACK, QCoreApplication::translate("UtilUI", SETTINGS_SOUNDS_PACK_DEFAULT)).toString() == QCoreApplication::translate("UtilUI", "Default"))`

Otherwise you'll compare "Default" to translated "Default.

Reply:
I tried but the result is same, migration isn't apply :(

Reply:
The problem is that the translation file is not loaded when `migrateSettings()` is called. Therefore `QCoreApplication::translate("UtilUI", "Default")` returns "Default".

I suggest to remove `SETTINGS_SOUNDS_PACK` from settings in `migrateSettings()` or leave it untouched.

Reply:
Ah yes, you're right! Thanks, I've removed it from migrateSettings().